MAUI/Circular-Charts/RadialBarChart.md

```diff
@@ -0,0 +1,433 @@
+---
+layout: post
+title: Radial Bar chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about radial bar chart and its features in Syncfusion .NET MAUI Chart Chart (SfCircularChart) control.
+platform: maui
+control: SfCircularChart
+documentation: ug
+---
+
+# Radial Bar Chart in .NET MAUI Chart
+
+[RadialBarSeries]() is a type of doughnut chart that represents each segment as a separate circle. It is used to compare values between various categories. To render a [RadialBarSeries]() in circular chart, create an instance of the [RadialBarSeries]() and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html#Syncfusion_Maui_Charts_SfCircularChart_Series) collection property of [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html).
+
+N> The circular chart has [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html#Syncfusion_Maui_Charts_SfCircularChart_Series) as its default content.
+
+The following properties can be used to customize the appearance of the radial bar segment:
+
+ * [Opacity](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_OpacityProperty) - To control the transparency of the chart segments.
+
+ * [Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CircularSeries.html#Syncfusion_Maui_Charts_CircularSeries_Stroke) - To customize the outer layer of the chart segments.
+
+ * [StrokeWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CircularSeries.html#Syncfusion_Maui_Charts_CircularSeries_StrokeWidth) - To customize the width of the stroke in chart segments.
+
+ * [GapRatio]() - To customize the spacing between each chart segments.
+
+ * [MaximumValue]() - To represent the span of the segment-filled area in the radial bar track. The default value of this property is `double.NaN`.
+
+ * [PaletteBrushes](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_PaletteBrushes) - To customize the appearance of the series.
+
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" />
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart type in MAUI Chart](Chart-Types_images/maui_radialbar_chart.png)
+
+## Changing the radial bar size
+
+You can use the [Radius]() property to change the radial bar chart size. The default value of the radius is [0.8]().
+
```

Comment:
Add link for Radius. 
Default value 0.8 syntax is wrong.

Reply:
I have added the radius property link and added proper syntax.